monkey/monkey_island/cc/resources/aws_exporter.py

```diff
@@ -0,0 +1,607 @@
+                "Description": "Use micro-segmentation policies to disable communication other than the required.",
+                "Remediation": {
+                    "Recommendation": {
+                        "Text": "Machines are not locked down at port level. Network tunnel was set up from {0} to {1}"
+                            .format(issue['machine'], issue['dest'])
+                    }
+                }}
+
+        if 'aws_instance_id' in issue:
+            finding["Resources"] = [{
+                "Type": "AwsEc2Instance",
+                "Id": instance_arn.format(instance_id=issue['aws_instance_id'])
+            }]
+        else:
+            finding["Resources"] = [{'Type': 'Other'}]
+
+        return finding
+
+    @staticmethod
+    def _handle_sambacry_issue(issue, instance_arn):
+        finding = \
+            {"Severity": {
+                "Product": 10,
+                "Normalized": 100
+            }, "RecordState": "ACTIVE", "Title": "Samba servers are vulnerable to 'SambaCry'",
+                "Description": "Change {0} password to a complex one-use password that is not shared with other computers on the network. Update your Samba server to 4.4.14 and up, 4.5.10 and up, or 4.6.4 and up." \
+                .format(issue['username']), "Remediation": {
+                "Recommendation": {
+                    "Text": "The machine {0} ({1}) is vulnerable to a SambaCry attack. The Monkey authenticated over the SMB protocol with user {2} and its password, and used the SambaCry vulnerability.".format(
+                        issue['machine'], issue['ip_address'], issue['username'])
+                }
+            }}
+
+        if 'aws_instance_id' in issue:
+            finding["Resources"] = [{
+                "Type": "AwsEc2Instance",
+                "Id": instance_arn.format(instance_id=issue['aws_instance_id'])
+            }]
+        else:
+            finding["Resources"] = [{'Type': 'Other'}]
+
+        return finding
+
+    @staticmethod
+    def _handle_smb_pth_issue(issue, instance_arn):
+        finding = \
+            {"Severity": {
+                "Product": 5,
+                "Normalized": 100
+            }, "RecordState": "ACTIVE",
+                "Title": "Machines are accessible using passwords supplied by the user during the Monkey's configuration.",
+                "Description": "Change {0}'s password to a complex one-use password that is not shared with other computers on the network.".format(
+                    issue['username']), "Remediation": {
+                "Recommendation": {
+                    "Text": "The machine {0}({1}) is vulnerable to a SMB attack. The Monkey used a pass-the-hash attack over SMB protocol with user {2}.".format(
+                        issue['machine'], issue['ip_address'], issue['username'])
+                }
+            }}
+
+        if 'aws_instance_id' in issue:
```

Comment:
This whole part (from `if 'aws_instance_id' in issue:` to `finding["Resources"] = [{'Type': 'Other'}]` is just copy-pasted a lot. Export to function

Reply:
Also, since this appears in all findings (or at least most of them), make the call to it from the new function which creates the finding (The one from the comment above)